dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/zk/ZKServer.java

```diff
@@ -108,11 +127,11 @@ public static void startLocalZkServer(final int port) {
      * @param tickTime    zk tick time
      * @param maxClientCnxns    zk max client connections
      */
-    private static synchronized void startLocalZkServer(final int port, final String dataDirPath,final int tickTime,String maxClientCnxns) {
-        if (zkServer != null) {
+    private void startLocalZkServer(final int port, final String dataDirPath,final int tickTime,String maxClientCnxns) {
+        if (zooKeeperServerMain != null) {
```

Comment:
Hi,

Is it better to add `isStarted.get()` compare in if statement?

Reply:
solved